doc/manual_install.md

```diff
@@ -7,6 +7,13 @@
 
 Compilation is only tested with very recent stable versions of `rustc`. If you use a distro with older Rust packages, consider using [rustup](https://rustup.rs/) to install a newer toolchain.
 
+On systems using alternative (non-glibc) C standard libraries like `musl`, `cargo` must be configured to be able to link it statically. Otherwise, blocks needing to link to system libraries like `temperature`, `sound` (for pulseaudio) and maybe others will not be usable due to segmentaion faults. To configure `cargo`, just add this to your `~/.cargo/config.toml`:
```

Comment:
I believe this config _disables_ static linking.

Reply:
Yes of course, that makes more sense. I was confused by the flag, but apparently the target flags start with `+` or `-` depending on whether they enable or disable, it's not a gnu style cli option